zScanner/Coordinators/NewDocumentCoordinator.swift

```diff
@@ -136,19 +136,19 @@ class NewDocumentCoordinator: Coordinator {
         pop()
     }
     
-    private func savePagesToDocument(_ pages: [UIImage]) {
+    private func savePagesToDocument(_ pages: [PageDomainModel]) {
 
         // Store images
         pages
             .enumerated()
-            .forEach({ (index, image) in
-                let page = PageDomainModel(image: image, index: index, correlationId: newDocument.id)
+            .forEach({ (index,page) in
```

Comment:
Check the formatting, missing space after the comma. It's a very small detail, but it will just look better :)